api/metrics-api-jaxrs/src/main/java/org/hawkular/metrics/api/jaxrs/MetricsServiceLifecycle.java

```diff
@@ -225,11 +228,10 @@ private void startMetricsService() {
             metricsService.setDateTimeService(createDateTimeService());
             metricsService.setDefaultTTL(getDefaultTTL());
 
-            // TODO Set up a managed metric registry
-            // We want a managed registry that can be shared by the JAX-RS endpoint and the core. Then we can expose
-            // the registered metrics in various ways such as new REST endpoints, JMX, or via different
-            // com.codahale.metrics.Reporter instances.
-            metricsService.startUp(session, keyspace, false, false, new MetricRegistry());
+            MetricRegistry metricRegistry = MetricRegistryProvider.INSTANCE.getMetricRegistry();
+            jmxReporter = JmxReporter.forRegistry(metricRegistry).inDomain("hawkular.metrics").build();
```

Comment:
Can we make this configurable? So that it does not start unless it is configured to start?